model/migrations/88cf753-conversation_state.sql

```diff
@@ -0,0 +1,14 @@
+--
+-- Table structure for table `conversation_state`
+--
+DROP TABLE IF EXISTS `conversation_state`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `conversation_state` {
+  `conversationId` varchar(255) COLLATE utf8mb4_bin not NULL,
```

Comment:
Wait I just realized we're missing the userId in this table, and the conversationId should be called uniqueId to conform to the dbproxy API.